zapier/introduction/what-is-zapier/zapier-common-terminology.md

```diff
@@ -0,0 +1,57 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Zapier Dependencies
+
+---
+## Content
+
+Now that you have your account, you are ready to start using Zapier.
```

Comment:
I'd remove this:
- it's not relevant if we look at this insight standalone
- was already mentioned twice in the previous insight